### PR TITLE
code cleanup

### PR DESCRIPTION
**Pull request by Amelia0010**
replaces var/global with GLOBs, and removes unused ones like religion_name

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -50,9 +50,6 @@ datum
 		var
 			varname1 = 1
 			varname2
-			static
```
Comment (Amelia0010) [inline .github/CONTRIBUTING.md:53]: I know this is just a text file, but I don't want us to encourage doing something that you're not supposed to do

**Comment by Hopekz** [comment]
Will this cause variables to escape their scope? 
I thought the point of the static var's was so that it would not apply to other scopes.

If not I have to modify my current project since I use static var's

**Comment by Amelia0010** [comment]
static vars are global vars, apparently :)

**Comment by alexkar598** [comment]
Static vars are globals but they are still limited to a scope

**Pull request merged**